Cargo.toml

```diff
@@ -13,6 +13,7 @@ description = "Adwaita-like SCTK Frame"
 smithay-client-toolkit = "0.16"
 tiny-skia = { version = "0.7.0", features = ["std", "simd"] }
 log = "0.4"
+memmap2 = "0.5.8"
```

Comment:
memmap2 is already a dependency of _smithay-client-toolkit_ so it adds no additional bloat using it here. I also didn't bother gating it on the `ab_glyph` feature because of this.